pvHelpers/email/email_v2.py

```diff
@@ -97,15 +102,19 @@ def toBrowser(self, with_body=False):
         o["flags"] = self.flags
         o["subject"] = self.subject
         # This needs fixing, should get the names from server
-        o["sender"] = {"address": self.sender["user_id"], "name": self.sender["display_name"]}
-        o["tos"] = [{"address": to["user_id"], "name": to["display_name"]} for to in self.tos]
-        o["ccs"] = [{"address": cc["user_id"], "name": cc["display_name"]} for cc in self.ccs]
-        o["bccs"] = [{"address": bcc["user_id"], "name": bcc["display_name"]} for bcc in self.bccs]
+        if self.sender.has_key("external_email"):
```

Comment:
We use a bunch of different ways to check if a key exists in a dict. It would be good if we settled on one.